### ISSUE TITLE
New azure storage libraries

### ISSUE THREAD

**Issue by nvmkpk** [Azure/azure-webjobs-sdk#2177]
Newest version of [WindowsAzure.Storage](https://www.nuget.org/packages/WindowsAzure.Storage/9.3.3) nuget package description says:
```
NOTE: As of version 9.4.0, this library has been split into multiple parts and replaced:  See https://www.nuget.org/packages/Microsoft.Azure.Storage.Blob/, https://www.nuget.org/packages/Microsoft.Azure.Storage.File/, https://www.nuget.org/packages/Microsoft.Azure.Storage.Queue/, and https://www.nuget.org/packages/Microsoft.Azure.Storage.Common/.
      For table support, see https://www.nuget.org/packages/Microsoft.Azure.CosmosDB.Table/.
      This client library enables working with the Microsoft Azure storage services which include the blob and file service for storing binary and text data, the table service for storing structured non-relational data, and the queue service for storing messages that may be accessed by a client.
      For this release see notes - https://github.com/Azure/azure-storage-net/blob/master/README.md and https://github.com/Azure/azure-storage-net/blob/master/changelog.txt
      Microsoft Azure Storage team's blog - http://blogs.msdn.com/b/windowsazurestorage/
```
When can I expect this sdk to start referencing these newer nuget packages?

**Comment by jeffhollan** [comment]
@fabiocav I wonder if it's time for us to just do a quick pass on our bindings and rev them to the latest.  There [was an issue today](https://github.com/MicrosoftDocs/azure-docs/issues/26479) where the user was attempting to use a newer version of Storage and hit exceptions.

**Comment by Miggleness** [comment]
I hit an issue too after referencing `Microsoft.Azure.EventHubs.Processor v3.0.0`

**Comment by jeffhollan** [comment]
Yes the event hubs work is being tracked here https://github.com/Azure/azure-webjobs-sdk/issues/2155

**Comment by fabiocav** [comment]
@jeffhollan reviewing the P1 assignments and, because of how long this has been in the queue, I'm removing that assignment so we can review.

**Comment by ericsampson** [comment]
> @fabiocav  reviewing the P1 assignments and, because of how long this has been in the queue, I'm removing that assignment so we can review.

What does that mean?

**Comment by AgSync-Aaron** [comment]
The `Microsoft.Azure.WebJobs.Host.Storage` library is NetStandard 2.0, but has a dependency on `WindowsAzure.Storage` which is **_NOT_** NetStandard 2.0. This is making an ugly dependency graph when trying to use it in a netcoreapp3.0 console application. I hope this can be tackled soon to use the newer Azure storage libraries as @nvmkpk pointed out 7 months ago.
![image](https://user-images.githubusercontent.com/43680940/68500410-abf38780-0229-11ea-86c4-360d7253dd36.png)

**Comment by albe** [comment]
In the meantime, there is storage sdk v12 https://github.com/Azure/azure-storage-net - so maybe it's not even worth adopting v11 by now and having to support that for some time

**Comment by MatthewSteeples** [comment]
Related (possibly duplicate) of #2065

**Comment by nvmkpk** [comment]
May be related but certainly not duplicate. Original request was to move to newer than 9.4 but v12 has yet another API rewrite. I personally stopped using this library and implemented a similar one myself.
